spec/support/active_record.rb

```diff
@@ -19,9 +20,17 @@
   create_table :cities do |t|
     t.column :country_id, :integer
     t.column :name, :string
+    t.column :surname, :string
```

Comment:
I will propose a not so important stuff: what about calling this new column `mayor_name` or something else?

It felt a little odd to me a city having surname (I also would not be impressed if there are cities somewhere with surnames...)

Reply:
You're right :)